.github/workflows/labeler.yml

```
@@ -0,0 +1,18 @@
+name: "Pull Request Labeler"
+
+on:
+  pull_request_target:
```

what is the value of the `"pull_request_target" => ?` association here? 

guessing  it it an empty map, consider:

```suggestion
  pull_request_target: {}
```

---

also, shouldn't we limit this to `types: [open]`? let's allow humans to unlabel something, for example if only `docs/READE` is changed.

---

Anything in the docs tree is docs related .. even if it is just the readme or the pom or whatever .. but yes, we should also allow unlabelling